modules/javafx.graphics/src/main/java/javafx/scene/SpotLight.java

```diff
@@ -125,9 +140,12 @@ protected void invalidated() {
 
 
     /**
-     * The angle of the spotlight's inner cone. Surfaces whose angle to the light's origin is less than this angle
-     * receive the full light's intensity. At larger angles, the light intensity starts to drop. The valid range is
-     * {@code 0 <= innerAngle <= outerAngle}; values outside of this range can produce unexpected results.
+     * The angle of the spotlight's inner cone, in degrees (as shown in the class doc image). A point whose angle to the
+     * light's position is less than this angle is not attenuated by the spotlight factor ({@code spot = 1}). At larger
```

Comment:
The term `angle to the light position` isn't precise since you need a second vector to form an angle. In the class docs, you use the wording `point whose vector to the light position forms an angle {@code theta} with the direction of the light`, which I like. Maybe you could work that into this description somehow?

Reply:
It can become cumbersome. Something like `A point whose whose vector to the light's position and the direction of the light form an angle that is less than this angle is not attenuated by the spotlight factor`  or `A point whose vector to the light's position forms an angle {@code theta} with the direction of the light, and {@code theta} is smaller than this angle, is not attenuated by the spotlight factor.` is a lot to digest.

I think that the meaning is intuitive. Maybe I can defined what an "angle of a point to the light" is in the class doc and use that definition in the `innerAngle` and `outerAngle` docs.

Reply:
Yeah, I see your point. And yes, I think it would help to define that term in the class docs.